src/encrypted_container.rs

```diff
@@ -0,0 +1,248 @@
+use serde::{Deserialize, Serialize};
+use trussed::types::{KeyId, Message};
+use trussed::{cbor_deserialize, cbor_serialize, try_syscall};
+
+/// The buffer size for the serialization operation of a single encrypted+serialized credential
+/// The minimum size should be about 256 (the current maximum packet length)
+///     + CBOR overhead (field names and map encoding) + encryption overhead (12B nonce + 16B tag)
+/// The extra bytes could be used in the future, when operating on the password-extended credentials.
+const SERIALIZED_OBJECT_BUFFER_SIZE: usize = 1024;
```

Comment:
As we already have to allocate a buffer for `Message`, I don’t think we need a separate serialization buffer – see the other comments regarding `Message::try_from`.

Reply:
Removed, with docs moved to general one.